README.md

```diff
@@ -1253,11 +1321,13 @@ But this window is not always open.
     the nuances of this before joining companies in the past, but with the risks of short
     exercise windows for employees becoming more widely known, the recent trend has shifted to
     longer exercise windows.
-    As an employee or a founder, it is both fair and wise to understand and negotiate these
+    As an employee or a founder, it is both fair and wiser to understand and negotiate these
```

Comment:
Why did "wise" get changed back to "wiser"? Wiser than what? Try "fairer and wiser" or "fair and wise"

Reply:
Was a reviewer suggestion. Will update.